docs/designs/conceptual_localization.md

```diff
@@ -0,0 +1,253 @@
+# Conceptual Localization(Loc) Support
+
+## Description
+
+DocFX build supports localization contents, there are a few features need to be supported for localization contents:
+  - Partial contents:
+    - The localization contents are often a part of source content.
+    - There maybe extra localization content which doesn't exist in source repo.
+  - Delay localization, which means the version of loc content are usually behind of source content version.
+  - Localization contents are not required to be stored in the same repo of source content, they can be anywhere.
+  - Localization publishing may have a few specific requirements which are different with source publishing:
+    - Most of the localization configuration would be same with source configuration, a few are different, set based on scenarios, so we don't know the exact set of different configurations.
+    - They may applies different configuration per branches.
+## Design
+
+- Treats localization content as the replacing content of source, they are not independent repo for publishing but only stores the corresponding loc content.
+- Localization publishing mixes the loc content and source content, loc content has higher priority to replace the source content.
+- Localization publishing uses source configuration and localization **overwrite** configuration
+
+## Workflow
+
+![Localization Publishing And Build](images/loc_build_publish.PNG)
+
+### Loc Content as Assets of Source Repo 
+
+  - No more specified fallback features, there is always mixed (Loc + Source) full set content for build/publishing  
+  - All features are equal to Source + Loc, less Loc repo specified features  
+  - Less Loc configurations to maintain and configuration changes in source repo will immediately be applied to Loc  
+  - Loc content can be stored in any places(one repo or multiple repo), what we need is the **mappings between Source content and Loc content**  
+  
+Below kinds of mappings are considered to be supported and there is a **strong convention** between source repo and loc repo:
+
+  - **Folder**, localization files are stored in the **same repository** with source files but under different **locale folder**
+    ```txt
+    source file         -->         localization files
+    /readme.md          -->         /localization/zh-cn/readme.md
+    /files/a.md         -->         /localization/de-de/files/a.md
+    ```
+  - **Repository**, localization files are stored in an **independent repository** per locale but keep the **same folder structure**
+      
+    Here is an string convention for loc repo name:
+    
+      - `{source-repo-name}` -> `{source-repo-name}.{locale}`
+      - `{source-repo-name}.{source-locale}` -> `{source-repo-name}.{loc-locale}`
+    
+    ```txt
+    source repo       ->locale->      localization repo
```

Comment:
this table needs to have 4 columns: {source-repo}, {source-locale}, {locale}, {localization-repo}. The terms used in the table is not the same as the term used in the convention. This creates questions like "is locale source-locale or loc-locale"?

Reply:
i think it's pretty clear for me since the below has some examples